addon-test-support/audit.js

```diff
@@ -66,10 +66,41 @@ Ember.Test.registerAsyncHelper('a11yAudit', function(app, ...args) {
  * @method a11yAudit
  * @public
  */
-export default function a11yAudit(...args) {
+function a11yAudit(...args) {
   if (window.a11yAudit) {
     return window.a11yAudit(...args);
   }
 
   return runA11yAudit(...args);
 }
+
+/**
+ * A wrapper method to run the a11yAudit if desired
+ *
+ * @method a11yAuditIf
+ * @public
+ */
+function a11yAuditIf(...args) {
+  if(getUrlParameter('enableA11yAudit') === 'true') {
```

Comment:
Wanted to cache the output of `getUrlParameter('enableA11yAudit')` however phantom keeps complaining 
```
Error: Browser disconnected
```